tests/main/test_main.py

```
@@ -731,6 +731,68 @@ def test_generate_with_use_annotated(output_dir: Path) -> None:
+        output_dir,
+        None,
+        enable_faux_immutability=True,
+        disable_timestamp=True,
+    )
+
+    models = output_dir.joinpath("models.py").read_text(encoding="utf-8")
+    assert "ConfigDict" in models
+    assert "frozen=True" in models
+
+    namespace: dict[str, Any] = {}
+    exec(models, namespace)
```

Ruff (0.15.12)

[error] Use of `exec` detected

(S102)